test/e2e/performanceprofile/functests/2_performance_update/updating_profile.go

```diff
@@ -831,6 +833,387 @@ var _ = Describe("[rfe_id:28761][performance] Updating parameters in performance
 			}
 		})
 
+		It("[test_id:50964] Offline Higher CPUID's", func() {
+			var offlineCPUs, reservedCpus, isolatedCpus string = "", "", ""
+			//This map is of the form numaNode[core][cpu-siblings]
+			var numaCoreSiblings map[int]map[int][]int
+			var onlineCPUInt int
+			for _, node := range workerRTNodes {
+				onlineCPUCount, err := nodes.ExecCommandOnNode([]string{"nproc", "--all"}, &node)
+				Expect(err).ToNot(HaveOccurred())
+				onlineCPUInt, err = strconv.Atoi(onlineCPUCount)
+				Expect(err).ToNot(HaveOccurred())
+				if onlineCPUInt <= 8 {
+					Skip(fmt.Sprintf("This test needs more than 8 CPUs online to work correctly, current online CPUs are %s", onlineCPUCount))
+				}
+			}
+			sep := ""
+			for _, node := range workerRTNodes {
+				numaCoreSiblings, err = nodes.GetCoreSiblings(&node)
+			}
+			for key := range numaCoreSiblings {
+				cores := make([]int, 0)
+				for k := range numaCoreSiblings[key] {
+					cores = append(cores, k)
+				}
+				sort.Ints(cores)
+				higherCoreIds := cores[len(cores)-1]
+				getCPUs(numaCoreSiblings, higherCoreIds, &offlineCPUs, sep)
+				sep = ","
+			}
+			sep = ""
```

Comment:
this also could be cleaner and outside the main test.

Reply:
addressed in the latest commit